src/um/wia.rs

```diff
@@ -0,0 +1,756 @@
+// Copyright © 2015-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms
```

Comment:
Copyright headers have been removed from winapi.

Reply:
Also this module should be renamed to `wia_lh` as the Windows SDK has all the definitions in `wia_lh.h` instead of `wia.h`.

Reply:
Thanks. The `Wia.h` file just has a switch to include either `wia_xp.h` (Windows xp and below) or `wia_lp.h` (modern Windows), so it is imported in C via "Wia.h". 

Should the file (and the feature etc.) be renamed to "wia_lh" instead, what do you suggest?  I understand that this crate is meant to mirror the files in the Windows SDK 1-to-1, so should users be using `wia_lh` in imports instead in this case?

Reply:
Yes, the module and features should be renamed to `wia_lh` and users should be importing through `wia_lh`.

Reply:
Thanks, fixed